Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ghcr.io/rse-ops/cuda-ubuntu-20.04:cuda-11.0.3
```

Comment:
To keep the top directory (relatively) clean, is it possible to move this file to `config/` or maybe `config/docker/`?